README.md

```diff
@@ -16,7 +16,17 @@ Then use the orb in a workflow.
 
 Orbs can be used with semantic version pins - see [CircleCI documentation](https://circleci.com/docs/2.0/creating-orbs/#semantic-versioning-in-orbs)
 
-### Version 1.1.2
+### Version 2.2.0
+
+The sous-chef standard for running platform tests is `kitchen/dokken-single` for every platform
+and each test should require `lint_and_unit` to allow circle to perform parallel builds.
+
+To generate your circle yaml config utilize:
+
+1. Alias `dokken` to `KITCHEN_LOCAL_YAML=kitchen.dokken.yml kitchen`
```

Comment:
Use `-` instead of a numbered list, this looks a bit weird

Reply:
this is a set of steps, so I think it makes more sense to have numbers rather than a non-ordered list

Reply:
🤔 ai true, except in markdown, they're all number 1) 1) 1) which doesn't make sense without a markdown previewer

Reply:
markdown lint doesn't like it when you actually use the numbers vs 1111 😂